examples/javascript/blob_upload_download_webapi.html

```diff
@@ -241,6 +243,15 @@ <h2>Blob Upload</h2>
               </div>
             </div>
 
+            <div class="col-12">
+              <label for="send-to-input" class="form-label">Send to</label>
+              <input id="send-to-input" value="0x9d8a07f9f5cb4d10ad3a8c949f17b9ae6c51fc76c2b7ef62dc5babec551e79e6" placeholder="Send to" class="form-control"
+                required />
+              <div class="form-text">
+                The number of Walrus epochs for which to store the blob.
```

Comment:
This needs to be updated:
```suggestion
                The address to which the newly created `Blob` object should be sent (optional).
```